src/tools/clippy/tests/ui/unnecessary_clone.stderr

```diff
@@ -75,13 +81,13 @@ LL |         let _ = &mut encoded.clone();
 help: try dereferencing it
    |
 LL |         let _ = &mut &(*encoded).clone();
-   |                      ~~~~~~~~~~~~~~~~~~~
+   |                      +++       ~~~~~~~~~
 help: or try being explicit if you are sure, that you want to clone a reference
    |
 LL |         let _ = &mut <&[u8]>::clone(encoded);
-   |                      ~~~~~~~~~~~~~~~~~~~~~~~
+   |                      +++++++++++++++       ~
 
-error: using `clone` on a double-reference; this will copy the reference of type `&[u8]` instead of cloning the inner type
+error: using `clone` on a double-reference, which copies the reference of type `[u8]` instead of cloning the inner type
```

Comment:
This seems to still test the `clone_double_ref` lint inside the Clippy UI tests. All tests that are there explicitly for this lint should be moved to rustc UI tests.